lms/djangoapps/grades/tests/test_signals.py

```diff
@@ -218,23 +218,26 @@ def test_raw_score_changed_score_deleted_optional(self):
         self.signal_mock.assert_called_with(**expected_set_kwargs)
 
     @ddt.data(
-        ['score_set', 'lms.djangoapps.grades.signals.handlers.submissions_score_set_handler',
+        ['score_set', SUBMISSION_KWARGS[SUBMISSION_SET_KWARGS]['points_earned'],
          SUBMISSION_SET_KWARGS],
-        ['score_reset', 'lms.djangoapps.grades.signals.handlers.submissions_score_reset_handler',
+        ['score_reset', 0,
          SUBMISSION_RESET_KWARGS]
     )
     @ddt.unpack
-    def test_disconnect_manager(self, signal_name, handler, kwargs):
+    def test_disconnect_manager(self, signal_name, weighted_earned, kwargs):
         """
         Tests to confirm the disconnect_submissions_signal_receiver context manager is working correctly.
         """
         signal = self.SIGNALS[signal_name]
         kwargs = SUBMISSION_KWARGS[kwargs].copy()
-        handler_mock = self.setup_patch(handler, None)
+        handler_mock = self.setup_patch('lms.djangoapps.grades.signals.handlers.PROBLEM_WEIGHTED_SCORE_CHANGED.send',
+                                        None)
 
         # Receiver connected before we start
         signal.send(None, **kwargs)
         handler_mock.assert_called_once()
```

Comment:
This assertion didn't exist in mock 1.0.1, upgrading revealed some flaws in the test setup.  The `receiver` decorators make it tricky to mock the signal handlers themselves, so mocked the main function they call instead.